channels/stable-4.6.yaml

```diff
@@ -1,5 +1,21 @@
 name: stable-4.6
 versions:
+- 4.5.18
```

Comment:
We need to hold these until we provide upgrades to 4.6. Lets drop these and ensure that our PR to bring upgrades to stable-4.6 includes them?

Reply:
Fixed with 6408ba1 -> 1d9fe0f.